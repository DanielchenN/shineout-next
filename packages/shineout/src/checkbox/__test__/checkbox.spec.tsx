import { render, cleanup } from '@testing-library/react';
import Checkbox from '..';
import mountTest from '../../tests/mountTest';
import CheckboxBase from '../__example__/001-base';

// const SO_PREFIX = 'checkbox'
afterEach(cleanup);
describe('Checkbox[Base]', () => {
  mountTest(Checkbox);
  test('should render correctly', () => {
    const { container } = render(<CheckboxBase />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
