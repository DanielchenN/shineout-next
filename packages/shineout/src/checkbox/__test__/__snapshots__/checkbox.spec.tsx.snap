// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox[Base] should render correctly 1`] = `
<div
  class="checkbox-wrapper-0-2-1"
>
  <input
    style="display: none;"
    tabindex="0"
    type="checkbox"
  />
  <div
    class="checkbox-indicatorWrapper-0-2-6"
  >
    <i
      class="checkbox-indicator-0-2-7"
      tabindex="1"
    />
  </div>
  <span
    class="checkbox-desc-0-2-8"
  >
    Checkbox
  </span>
</div>
`;
